### PR TITLE
Fix libjq not found by the linker

### PR DESCRIPTION
**Pull request by terry-xiaoyu**

### CODE REVIEW

**Comment by kjellwinblad** [comment]
Is there a way to also statically link on MacOS?

**Comment by kjellwinblad** [comment]
@terry-xiaoyu  FYI, I have merged a PR that puts the version number in the name of the shared library for the NIF (this is necessary for hot upgrading). I think you should rebase this PR on the new main branch since there might be conflicts with this PR.

File: c_src/get_version.sh [hunk 1]
```diff
@@ -1,4 +1,3 @@
-#!/bin/sh
+#!/usr/bin/env bash
 
-
-grep '.*define.*(.*VERSION' "$1"/src/jq.erl | sed -e 's/.*VERSION.*,.*\([[:digit:]]\+\).*/\1/'
+grep '.*define.*(.*VERSION' "$1"/src/jq.erl | sed -E 's/.*VERSION.*,.*([[:digit:]]+).*/\1/'
```
Comment (kjellwinblad) [inline c_src/get_version.sh:3]: Does this script depend on any bash feature? If not, it seems unnecessary to add bash as a dependency.

**Review by kjellwinblad — CHANGES_REQUESTED** [review]
Looks good except for the two comments I have made. Also, can you do a rebase (not merge) on the main branch and squash the commits in this PR before we merge it?

File: c_src/get_version.sh (hunk 1 above)
Comment (terry-xiaoyu) [reply c_src/get_version.sh:3]: I'll change it back to sh.

**Review by kjellwinblad — APPROVED** [review]
Looks good!

**Pull request merged**